### DISCUSSION TITLE
CSS issues with latest Curator v3 Beta

### DISCUSSION THREAD

**Discussion by pekka** [awcodes/filament-curator#285] [Q&A]
I've installed Curator into a Filament v3 project. The Media resource pages are working fine, but the picker is showing weird CSS glitches. Buttons are misaligned or covered by each other, and the images to select show with 100% with instead of as thumbnails:

<img src="https://github.com/awcodes/filament-curator/assets/262464/33f17bf2-6b41-4af5-8c60-80609e3d7b05" width=200>

What I've tried:
- Uninstalled and reinstalled the Curator package
- Made sure the CSS is correctly included (it is; I looked in the CSS the browser ends up receiving)
- Made sure cropper.js is installed
- Checked for any errors in the console (none there)

Does anyone have an idea where to start to debug this?

**Comment by awcodes** [comment]
Did you also include the views in your tailwind config? I think you skipped a step.

**Comment by 000yuying** [comment]
I'm encountering the same issue.
and tryed Uninstalled and reinstalled the Curator package

Please assist me.

<img width="751" alt="截圖 2023-10-03 15 50 42" src="https://github.com/awcodes/filament-curator/assets/109067769/adc3b6aa-1bae-4933-8f60-3625d234268e">
<img width="751" alt="截圖 2023-10-03 15 50 35" src="https://github.com/awcodes/filament-curator/assets/109067769/3fd527e7-3e21-4d7b-9d46-61c7153f23b9">
<img width="1440" alt="截圖 2023-10-03 15 54 03" src="https://github.com/awcodes/filament-curator/assets/109067769/6a2b0cdd-ca36-4fe3-8204-090b4b48704f">

**Reply by 000yuying** [reply]
Oh, I found my missing part.
I had skipped a step, which is adding ->viteTheme('resources/css/filament/admin/theme.css') inside AdminPanelProvider.
After adding it, I ran npm run build again,
and now everything is working fine.👍

**Reply by FilipRastovic** [reply]
i had the issue too, you need to follow the docs from curator about creating a custom theme, it's few commands from filament docs and then few things from curator docs too.
